snuba/cli/subscriptions_executor.py

```diff
@@ -31,16 +31,7 @@
     "entity_names",
     required=True,
     multiple=True,
-    type=click.Choice(
-        [
-            EntityKey.EVENTS.value,
-            EntityKey.TRANSACTIONS.value,
-            EntityKey.METRICS_COUNTERS.value,
-            EntityKey.METRICS_SETS.value,
-            EntityKey.GENERIC_METRICS_SETS.value,
-            EntityKey.GENERIC_METRICS_DISTRIBUTIONS.value,
-        ]
-    ),
+    type=click.Choice([entity_key.value for entity_key in EntityKey]),
```

Comment:
Agree that enumerating all of the entity keys is verbose, but there was a reason for that. It's because not all entities actually support subscriptions. Now we are allowing entities to be passed that we cannot actually have subscriptions on. I think I'd prefer to leave it as is.

Reply:
oh got it. will abandon this PR. thanks